utils/zerovec/src/map2d/cursor.rs

```diff
@@ -113,25 +115,6 @@ where
         })
     }
 
-    /// Transform this cursor into an ordered iterator over keys1 for a particular key0.
-    pub fn into_iter1(
```

Comment:
I removed this function because it doesn't do anything over `.iter1()` because we can't move values out of a ZeroVec.

Reply:
It does something, because I can be turning a temporary ZeroMap2d into something else, in which case I don’t want to lug the temporary lifetime around.
`iter1_copied` doesn’t work in my tests in #2883, because PatternULE is not Copy.